extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.css

```diff
@@ -40,4 +43,129 @@ amp-story-subscriptions.i-amphtml-story-subscriptions-visible::before {
 amp-subscriptions-dialog.i-amphtml-story-subscriptions-dialog-wrapper {
   border-radius: 16px 16px 0px 0px !important;
   font-family: Poppins, sans-serif !important;
+  font-weight: 400 !important;
+  background: white !important;
+  color: black !important;
```

Comment:
Please use hex values for color.

Reply:
Could you share why it is preferred? I think I have heard that the keyword is preferred but cannot find it in https://www.w3.org/TR/css-color-3/#colorunits. I do find it is acceptable in https://engdoc.corp.google.com/eng/doc/devguide/htmlcss/styleguide.md?cl=head#Hexadecimal_Notation.